inbm/log-changes.txt

```diff
@@ -1,2 +1,5 @@
 Changes in logging for inbm in next version
 -------------------------------------------
+### Fixed
+ - HSD 15011727901 - Fix POTA/INBC POTA print "Firmware Update Aborted" message after firmware update is success
```

Comment:
I don't think you need anything in this log file.  @gblewis1 - is that correct?  Since it's fixing a bug, validation knows the response will be different.

Reply:
@nmgaston ah, I missed this.. that seems fine

Reply:
I'll remove that in this branch and then we can merge.